locations/spiders/teds_montana_grill.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+
+class TedsMontanaGrillSpider(scrapy.Spider):
+    name = "teds_montana_grill"
+    allowed_domains = ["www.tedsmontanagrill.com"]
+    item_attributes = {"brand": "Ted's Montana Grill", "brand_wikidata": "Q16953170"}
+    start_urls = ("https://www.tedsmontanagrill.com/locations.html",)
+
+    def parse(self, response):
+        location_selectors = response.xpath('.//div[@class="location-list-item"]')
+        for location_selector in location_selectors[:]:
+            if location_selector.xpath(".//h3/text()").extract_first():
+                state = location_selector.xpath(".//h3/text()").extract_first()
+            yield from self.parse_location(location_selector, state)
+
+    def parse_location(self, location, state):
+        ref = location.xpath(".//a/@href").extract_first()
+        city = location.xpath(".//h5/text()").extract_first()
+        phone = (
+            location.xpath(".//p")[-1]
+            .xpath(".//span/a/@href")
+            .extract_first()
+            .replace("tel:", "")
+        )
+        address = location.xpath(".//p")[-1].xpath("text()").extract()
+        addr_full = ", ".join([line.strip() for line in address[:-1]])
```

Comment:
Could get the first line of this and set it as the `street_address` in addition to joining it into `addr_full`?

Reply:
Makes sense. It turns out that `addr_full` really was the `street_address`, so I have added the `street_address` field and included it with the `postcode` as `addr_full`.